### PR TITLE
Implemented parser for new PSL-based entities.

### PR DESCRIPTION
**Pull request by matijaSos**

### CODE REVIEW

File: waspc/src/Parser/EntityPSL.hs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Parser.EntityPSL
+    ( entityPSL
+    ) where
+
+import Text.Parsec.String (Parser)
+
+import qualified Wasp.EntityPSL
+import qualified Parser.Common as P
+import qualified Lexer as L
+
```
Comment (Martinsos) [inline waspc/src/Parser/EntityPSL.hs:10]: Maybe you could explain here what PSL stands for?

File: waspc/src/Wasp/EntityPSL.hs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+-- TODO(matija): remove PSL suffix, added it to avoid clashes with the existing Entity module.
+-- Once the old Entity module is removed, rename to "Entity".
+module Wasp.EntityPSL
+    ( EntityPSL (..)
+    ) where
+
+data EntityPSL = EntityPSL
+    { _name :: !String
+    , _pslModelSchema :: !String
```
Comment (Martinsos) [inline waspc/src/Wasp/EntityPSL.hs:9]: This should probably be Text? But ok no biggie.

File: waspc/src/Wasp/EntityPSL.hs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+-- TODO(matija): remove PSL suffix, added it to avoid clashes with the existing Entity module.
```
Comment (Martinsos) [inline waspc/src/Wasp/EntityPSL.hs:1]: Well ok, maybe we will have multiple ways to define Entities, so this is not so terrible actually, I like it.

**Review by Martinsos — APPROVED** [review]
Cool!

File: waspc/src/Parser/EntityPSL.hs (hunk 1 above)
Comment (matijaSos) [reply waspc/src/Parser/EntityPSL.hs:10]: right, good idea! Added explanation - it stands for Prisma Schema Language

File: waspc/src/Wasp/EntityPSL.hs (hunk 3 above)
Comment (matijaSos) [reply waspc/src/Wasp/EntityPSL.hs:1]: ok cool, might be nice

File: waspc/src/Wasp/EntityPSL.hs (hunk 2 above)
Comment (matijaSos) [reply waspc/src/Wasp/EntityPSL.hs:9]: you are right! switched to Text

**Pull request merged**